ci-operator/step-registry/observers/resource-watch/observers-resource-watch-observer.yaml

```diff
@@ -0,0 +1,10 @@
+observer:
+  name: observers-resource-watch
+  from: tests
+  commands: observers-resource-watch-commands.sh
+  resources:
+    requests:
+      cpu: 10m
```

Comment:
I would bump that up quite a bit, from my testing it looks like we sometimes struggle to commit to the repo fast enough, some of that is probably disk but I think we should err on the side of caution and a tenth of a CPU is very low. (this is 0.01 CPUs) I suggest just changing this to "1" for a full CPU, or maybe just "500m" for a half CPU.